src/smart/SmartPatient.js

```diff
@@ -140,6 +140,10 @@ export function SmartPatient() {
 }
 
 function cleanFsh(fsh) {
+  if (typeof fsh != "string") {
```

Comment:
This function can be rewritten as:

function cleanFsh(fsh) {
  const fshString = typeof fsh === "string" ? fsh : fsh.fsh;
  return fshString.replaceAll('undefined', '\n').replaceAll(',', '\n');
}

Reply:
Rewrote this in https://github.com/ccsm-cds-tools/ccsm-cds-dashboard/pull/5/commits/fc0a1a263b87eab894bc01d0e054b700de27ba35